.github/workflows/push.yml

```
@@ -0,0 +1,101 @@
+        with:
+          registry: europe-docker.pkg.dev
+          username: oauth2accesstoken
+          password: ${{ steps.gcp-auth.outputs.access_token }}
+
+      - name: Push ActivityPub Docker image to private registry
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: true
+          tags: ${{ inputs.activitypub-private-tags }}
+          labels: ${{ inputs.activitypub-private-labels }}
```

actionlint (1.7.7)

property "activitypub-private-labels" is not defined in object type {activitypub-private-tags: string; activitypub-public-tags: string; migrations-private-tags: string; migrations-public-tags: string}

(expression)